src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -14,59 +14,102 @@
 limitations under the License.
 -->
 
-<md-input-container class="md-block">
-  <label>Container image</label>
-  <input ng-model="ctrl.containerImage" required>
-</md-input-container>
-<md-input-container class="md-block">
-  <label>Number of pods</label>
-  <input ng-model="ctrl.replicas" type="number" required min="1">
-</md-input-container>
-<div layout="row" ng-repeat="portMapping in ctrl.portMappings">
-  <md-input-container flex>
-    <label>Port</label>
-    <input ng-model="portMapping.port" type="number" min="0">
+<kd-help-section>
+  <md-input-container class="md-block">
+    <label>Container image</label>
+    <input ng-model="ctrl.containerImage" required>
   </md-input-container>
-  <md-input-container flex>
-    <label>Target port</label>
-    <input ng-model="portMapping.targetPort" type="number" min="0">
+  <kd-user-help>
+    Enter the URL of a public image on any registry, or a private image hosted on Docker Hub or Google Container Registry
+    <a href=""> Learn More </a>
```

Comment:
No need to wrap "Lear More" with spaces. Just <a>Learn More</a>.
 Add a line note